call-center/server/controllers/calls.controller.ts

```diff
@@ -75,6 +75,59 @@ class CallsController {
     }
   };
 
+  // Transfer the call to an agent or phone number to join
+  public static transfer = async function (req: Request, res: Response) {
+    // TODO Move `telnyx` declaration to module import once issue is re-fixed:
+    // https://github.com/team-telnyx/telnyx-node/issues/26
+    let telnyx = telnyxPackage(process.env.TELNYX_API_KEY);
+    let { transfererSipUsername, to } = req.body;
```

Comment:
You can remove this TODO now that https://github.com/team-telnyx/telnyx-client-examples/pull/28 has been merged
```suggestion
    let { transfererSipUsername, to } = req.body;
```